presto-parquet/src/main/java/com/facebook/presto/parquet/predicate/TupleDomainParquetPredicate.java

```diff
@@ -280,6 +282,13 @@ else if (isVarcharType(type) && columnDescriptor.getType() == PrimitiveTypeName.
         return null;
     }
 
+    private static void failWithCorruptionException(boolean failOnCorruptedParquetStatistics, String column, ParquetDataSourceId id, Statistics statistics)
+    {
+        if (failOnCorruptedParquetStatistics) {
+            throw new ParquetCorruptionException(format("Corrupted statistics for column %s in Parquet file %s: %s", column, id, statistics));
```

Comment:
I hope all children of `Statistics` type have `toString` implemented.

Reply:
One minor change we can make to the error message is to add quotes/brackets to set the boundaries of the column/file name and stats information:

```
Corrupted statistics for column "column_name" in Parquet file "file_name": [stats_string]
```

Currently it looks like the following, which is a little bit hard to follow:

```
Corrupted statistics for column DateColumn in Parquet file testFile: min: 200, max: 100, num_nulls: 0
```